packages/ERTP/src/types.js

```diff
@@ -103,7 +103,8 @@
  * cannot use the brand by itself to verify that a purported amount is
  * authentic, since the brand can be reused by a misbehaving issuer.
  *
- * @property {(issuer: Issuer) => boolean} isMyIssuer
+ * @property {(allegedIssuer: any) => boolean} isMyIssuer Should be used with
+ * `issuer.getBrand` to ensure an issuer and brand match.
```

Comment:
I see that src/amountMath.js lines 80 and 92 that we went from typing an `allegedAmount` argument from `any` to `Amount`. Which makes sense, because we are *certainly* not using this type system to keep track of whether we're relying on these objects.
```suggestion
 * @property {(allegedIssuer: Issuer) => boolean} isMyIssuer Should be used with
 * `issuer.getBrand` to ensure an issuer and brand match.
```

However, above, you change the typing of the `allegedIssuer` argument from `Issuer` to `any` which goes exactly in the opposite direction. They cannot both be improvements. I'm happier with the types saying what things are supposed to be, rather than what we assume them to be, since 
   * we should never use the type system for the latter, and
   * we use the types primarily to improve the IDE experience when writing calls, which is definitely enhanced by seeing what it is supposed to be.

So this parameter should remain typed `Issuer`